ch5/Makefile

```
@@ -0,0 +1,3 @@
+init:
+	pip install -U pip
+	pip install -r requirements.txt
```

```suggestion
	pip install fastapi[all]==0.87.0
```
여기는 다른 챕터와의 통일감을 위해서 `requirements.txt` 대신에 이렇게 하면 좋을 것 같습니다~

---

넵 반영하겠습니다!